spacy/lang/grc/CONTRIBUTOR_AGREEMENT.md

```diff
@@ -0,0 +1,106 @@
+      a derivative work of your contribution, the one who makes the derivative
+      work (or has it made will be the sole owner of that derivative work;
+
+    * you agree that you will not assert any moral rights in your contribution
+      against us, our licensees or transferees;
+
+    * you agree that we may register a copyright in your contribution and
+      exercise all ownership rights associated with it; and
+
+    * you agree that neither of us has any duty to consult with, obtain the
+      consent of, pay or render an accounting to the other for any use or
+      distribution of your contribution.
+
+3. With respect to any patents you own, or that you can license without payment
+to any third party, you hereby grant to us a perpetual, irrevocable,
+non-exclusive, worldwide, no-charge, royalty-free license to:
+
+    * make, have made, use, sell, offer to sell, import, and otherwise transfer
+      your contribution in whole or in part, alone or in combination with or
+      included in any product, work or materials arising out of the project to
+      which your contribution was submitted, and
+
+    * at our option, to sublicense these same rights to third parties through
+      multiple levels of sublicensees or other licensing arrangements.
+
+4. Except as set out above, you keep all right, title, and interest in your
+contribution. The rights that you grant to us under these terms are effective
+on the date you first submitted a contribution to us, even if your submission
+took place before the date you sign these terms.
+
+5. You covenant, represent, warrant and agree that:
+
+    * Each contribution that you submit is and shall be an original work of
+    authorship and you can legally grant the rights set out in this SCA;
+
+    * to the best of your knowledge, each contribution will not violate any
+    third party's copyrights, trademarks, patents, or other intellectual
+    property rights; and
+
+    * each contribution shall be in compliance with U.S. export control laws and
+    other applicable export and import laws. You agree to notify us if you
+    become aware of any circumstance which would make any of the foregoing
+    representations inaccurate in any respect. We may publicly disclose your
+    participation in the project, including the fact that you have signed the SCA.
+
+6. This SCA is governed by the laws of the State of California and applicable
+U.S. Federal law. Any choice of law rules will not apply.
+
+7. Please place an “x” on one of the applicable statement below. Please do NOT
+mark both statements:
+
+    * [x] I am signing on behalf of myself as an individual and no other person
+      or entity, including my employer, has or will have rights with respect to my
+      contributions.
+
+    * [x] I am signing on behalf of my employer or a legal entity and I have the
+      actual authority to contractually bind that entity.
```

Comment:
Could you select either one of the two statements - but not both?

Also, this contributor statement should be moved to `.github/contributors` and renamed to `jmyerston.md`